opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -432,6 +432,12 @@ message IntHistogramDataPoint {
   // buckets if a histogram is provided.
   sfixed64 sum = 5;
 
+  // A histogram may optionally contain the distribution of the values in the
```

Comment:
Does this belong as a message-level comment?

Reply:
done.